### PR TITLE
Local config feature

### PR DESCRIPTION
**Pull request by Setnea**
Task #111

### CODE REVIEW

File: config/IPIQA.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<main>
+<CONFIG>
+    <!-- IPIQA configurations: -->
+    <parameter key="number_of_cores" value= "6"/>
+    <parameter key="experiment_finished" value="True"/>
```
Comment (stork119) [inline config/IPIQA.xml]: number_of_cores and number_of_cores don't describe parameters of computer, but rather execution. Please, remove them from IPIQA.xml

File: python/WorkflowEngine.py [hunk 2]
```diff
@@ -9,41 +9,76 @@ def main():
...
+    logger.debug('Input settings source: %s', args.s[0])
     if os.path.isabs(args.s[0]): #relative or absolute path
         settings_path = FM.path_unify(args.s[0])
     else:
-        settings_path = FM.path_join(PP_path, "configuration_settings" , args.s[0])
-    if args.m != False: #single or multiple settings
+        settings_path = FM.path_join(PP_path, 
+                                     'configuration_settings' ,
+                                     args.s[0])
+    if args.a != False: # optional argument
```
Comment (stork119) [inline python/WorkflowEngine.py:54]: @Setnea why we cannot write ?: 
```
if args.a:
    additional_arg = {(args.a[0]) : (args.a[1])}
``` 

https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations

File: python/WorkflowEngine.py [hunk 3]
```diff
@@ -9,41 +9,76 @@ def main():
...
+        additional_arg = {(args.a[0]) : (args.a[1])}
+    else:
+        additional_arg = {}
+    if args.c != False: # optional (additional) config
```
Comment (stork119) [inline python/WorkflowEngine.py:58]: same

**Review by stork119 — CHANGES_REQUESTED** [review]
Look at my doubts about style and IPIQA.xml

File: config/IPIQA.xml (hunk 1 above)
Comment (Setnea) [reply config/IPIQA.xml]: Fixed.

File: python/WorkflowEngine.py (hunk 2 above)
Comment (Setnea) [reply python/WorkflowEngine.py:54]: @stork119 Lol, you're right, silly me.

Edit:  Actually, not so silly, but tired (when commenting). This statement is right, due to a fact that args.a is not boolean, it is False ONLY when argument was not specified.

File: python/WorkflowEngine.py (hunk 3 above)
Comment (Setnea) [reply python/WorkflowEngine.py:58]: same

**Review by stork119 — APPROVED** [review]

**Comment by Setnea** [comment]
I'm gonna delete branch in the evening, I have to check out sth.

**Pull request merged**